### ISSUE TITLE
Alumni Student Panel Reflection

### ISSUE THREAD

**Issue by raunak2007** [raunak2007/csa-pages#3]
# Takeaways and Reflection from the Student Panel

As a current CSA student, I was overjoyed to see DNHS CS Alumni talk about their experiences in the world of computer science and how their interaction with CS fundamentals in CSP/CSA led them to gain internships, as well as admission to great CS/Engineering departments at major universities. Their discussion about applying for college enlightened me because it showed how competitive and rigorous CS departments at most schools (especially UCs) are and how CS can be lottery-based at universities such as UCSD. This is most likely going to motivate me to major in CS at UCs/other universities where it is mostly impossible to switch into CS (or is lottery-based), but then apply for data science at most higher-level universities (such as the Ivy League universities, Stanford, MIT, and Caltech) where I can switch into CS if I feel as if the courses related to my data science major aren't rigorous enough. In addition, if I apply for data science as my major to higher-level universities, it will most likely boost my chances of getting into that university, as CS is getting increasingly competitive these days. However, given what Anthony said about his experience at the UC Irvine CS department, the introductory courses for a CS major should be given that I got an A in all 3 trimesters of CSP and I'm on track to get an A this trimester of CSA. In addition, something else that motivated me during this initial part of the Student Panel was that UCSD CS professors were motivated to make their students create portfolios of their work. This motivated me because this is very similar to the Night at the Museum (N@tM) performance that DNHS CS Students do every trimester and because that could potentially double as a personal project that I could show to employers in the near future. In addition, I remember Mabel talking about the UCSD CS department making a big push towards UI/UX and collaborating with the UCSD Linguistics department with using ChatGPT. That deeply interested me because those two concepts have been emphasized a lot in CSP/CSA (the latter concept being heavily used and encouraged by Mortensen himself) and my relatively large experience with those two concepts could help me a lot if I end up going to UCSD to major in CS.

After that, the presenters started talking about getting internships and job opportunities related to CS. I was deeply inspired by Tristan's experience, as he was able to get a great CS internship from the father of his friend Akhil Nandhakumar by showing the passion projects that he worked on during CSP and CSA. This motivated me a lot because I am also a close friend of Akhil and would definitely be interested in an internship opportunity at his father's company. In addition, it also motivated me because Tristan never planned on going to a 4-year college but still was able to get an internship at a good company nonetheless, saving him tens of thousands of dollars. In addition, Colin and Mabel talked about their experience with the Northrop Grumman HIP internship and how their initial one-year experience with Cyber Patriot helped them get that internship. In addition, they talked about how it is important to apply to internships early (usually in the fall of the school year you are applying in) and how it is also important to indicate whether you will be continuing the internships with your employer/team next summer or not. This impacted me because I am currently in an internship with the UCSD CSE Ujima Security and Privacy Lab and was able to successfully cold email and receive the internship by showing my experience with Python/machine learning fundamentals in CSP. However, I will utilize the advice given by the presenters in the Student Panel today and show my experience in CSP/CSA, my experience in past/current internships, and my personal projects, and will apply early to all the internships I want to target this year to maximize my chances of getting into a top 10 CS undergrad department the year after that.  

After that, the presenters started talking about their personal projects and their personal websites. In addition to Tristan's experience with personal projects he created during his time in CSP/CSA, he talked about his personal website, which he said isn't great right now (even though I personally thought it was awesome) and which he is going to update to make it even better. I asked him a question about what to put on a personal website besides resume-related details, and he responded by saying that you can put aesthetics/CSS-related tangibles, as well as personal projects that you have worked on and can show your current experience with CS. In addition, Colin and Mabel talked about their personal projects relating to an online storefront for a thrift shop at Escondido. I found that inspiring because it somewhat motivated me to make a website for Bengali businesses in San Diego who don't have much outreach besides word-of-mouth communication, which could deeply impact my community, friends, and family. In addition, Anthony mentioned about one of his friends being fascinated by Valorant and wanting to create a better version of Valorant using his CS skills. This gave me inspiration to do the same with Discord, given that I use Discord a lot and it would majorly benefit my friends if they had access to a more convenient form of Discord.

In conclusion, I am very grateful that I had the opportunity to attend this Student Panel and I thank the presenters for coming to this event in the midst of their (most likely) relatively-busy schedules. In addition, I am thankful for the Girls in CS club for hosting this special event. I hope the takeaways I learned from this event will prove to be useful and successful in the near future.
